tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "ESNEXT" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
```

Comment:
You wanna do this instead of a more "reasonable" target?

Reply:
Suggestions?

Reply:
I usually see ES5 being used the most, so IE11 people don't explode on impact. (those poor people on IE)

Reply:
Updated to ES2018 - we can change later if needed